File: corpus/javascript/fixture_23.js
export function calculateDiscount23(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 230 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook23 {
  lookupPrice(sku) { return sku.length + 23; }
}

File: corpus/python/fixture_23.py
def calculate_discount_23(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 230:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook23:
    def lookup_price(self, sku):
        return len(sku) + 23
